docs/dm/attestation-results/attestation-results.cddl

```diff
@@ -10,6 +10,7 @@ attestation-results = {
   ? ar.raw-evidence => ar-bytes
   ar.timestamp => ar-timestamp
   ? ar.endorsed-claims => ar-endorsed-claims
+  ar.appraisal-policy-id => text
```

Comment:
Should we leave it just as a text or a type choice with restricted set ?
like a resource identifier or something similar?

Reply:
what do you mean exactly by "resource identifier"?

Reply:
I meant URI, here!

Reply:
The question is whether the receiver needs to make sense of the returned policy-id (in which case typing it makes sense), or it is just an opaque handler whose "resolution" is only known to the Verifier.

Reply:
Our intent was human consumption rather than machine so I would favour leaving it as text.

Reply:
Thanks for the clarification. Ok, understood the intent. Fine with me. I will mark the conversation as resolved!